src/components/WithInfoText/index.tsx

```diff
@@ -34,7 +34,7 @@ export const WithInfoText = ({
         {children}
 
         <NextLink {...buttonBaseProps} href={href}>
-          <Icon.InfoCircle />
+          <Icon.BookOpen01 />
```

Comment:
If we change the icon here, we should also change the component name.

Reply:
i'll keep the original